app/containers/App/index.js

```diff
@@ -5,9 +5,9 @@ import { Router, Route, Switch } from 'react-router-dom';
 import indexRoutes from 'routes/index';
 
 // eslint-disable-next-line import/extensions
-import './material-dashboard-pro-react.css?v=1.2.0';
+import './material-dashboard-pro-react.css';
 
-const hist = createBrowserHistory();
+const hist = typeof document !== 'undefined' ? createBrowserHistory() : {};
```

Comment:
What's the reason for changing this?

Reply:
Right, I get it. This is so that the tests pass right?

I think we should stick to what we do at Fairfax - we should not be testing containers. It's too hard to mock stuff.

**Especially** if we end up changing app logic so that the tests pass

Reply:
Oops, didn't mean to check this in. Reverting.